backend/tests/assetlinks.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from '@jest/globals';
+import request from 'supertest';
+import { Server } from 'http';
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+process.env.VIDEO_SERVICE_PROVIDER = 'opentok';
+
+const startServer = (await import('../server')).default;
+const fileName = fileURLToPath(import.meta.url);
+const dirName = path.dirname(fileName);
+
+describe('GET /.well-known/assetlinks.json', () => {
+  let server: Server;
+
+  beforeAll(async () => {
+    server = await startServer();
+  });
+
+  afterAll((done) => {
+    server.close(done);
+  });
+
+  it('returns a 200 status code', async () => {
+    const res = await request(server).get('/.well-known/assetlinks.json');
+    expect(res.statusCode).toEqual(200);
+  });
+
+  it('returns the correct Content-Type header', async () => {
+    const res = await request(server).get('/.well-known/assetlinks.json');
+    expect(res.headers['content-type']).toMatch(/application\/json/);
+  });
+
+  it('returns valid JSON content', async () => {
+    const res = await request(server).get('/.well-known/assetlinks.json');
+    expect(() => JSON.parse(res.text)).not.toThrow();
+  });
+
+  it('returns the same content as the static assetlinks.json file', async () => {
+    const res = await request(server).get('/.well-known/assetlinks.json');
+    const expectedContent = fs.readFileSync(
+      path.join(dirName, '..', '.well-known', 'assetlinks.json'),
+      'utf8'
+    );
+
+    expect(JSON.parse(res.text)).toEqual(JSON.parse(expectedContent));
```

Comment:
Same nit here.  Since we can't add comments to JSON files, maybe we can explain why the non-obvious fields have the values they do.

Reply:
much simpler, absolutely. done here https://github.com/Vonage/vonage-video-react-app/pull/202/commits/b699741469296686d003da9fd04b738b931bc781